greenpithumb/greenpithumb.py

```diff
@@ -114,7 +115,10 @@ def main(args):
             for current_poller in pollers:
                 current_poller.start_polling_async()
             while True:
-                record_processor.process_next_record()
+                record_processor.try_process_next_record()
+                time.sleep(0.5)
```

Comment:
It's a little too long though because 0.5 means that if 5 events happen at once, it'll take us 2.5 seconds to process. It probably would make sense to have `try_process_next_record` return `True` or `False` depending on if it processed anything and we only sleep when it's empty.

For now I'll lower to 0.1.